apps/api/src/app/events/usecases/send-test-email/send-test-email.usecase.ts

```diff
@@ -111,6 +112,10 @@ export class SendTestEmail {
 
       html = data.outputs.body;
       subject = data.outputs.subject;
+
+      if (data.providers && typeof data.providers === 'object') {
+        bridgeProviderOptions = data.providers[integration.providerId] || {};
```

Comment:
I like using the naming of `bridgeProviderData` everywhere, it's a bit more explicit.